swagger.yml

```diff
@@ -36,6 +36,9 @@ definitions:
       status-code:
         type: integer
         format: int32
+      response-time:
```

Comment:
Awesome - could we add a description to specify what format it is in, without having to look into the code ?

Reply:
I mean - the format as in `milliseconds` or `nanoseconds` or whatever we end up with.